<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="9" column="0">
      <widget class="QLabel" name="_mission_lng">
       <property name="text">
        <string>Mission Lng</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="_gps_mission_lng_box"/>
     </item>
     <item row="7" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="_gps_lat_box"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="_gps_lng">
       <property name="text">
        <string>Longitude</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="_gps_mission_lat_box"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="_gps_hMSL_box"/>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="_mission_enable_box">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CheckBox</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="_gps_lat">
       <property name="text">
        <string>Latitude</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="_gps_fixtype_box"/>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="_waypoint_bearing_box"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Bearing</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="_gps_gSpeed">
       <property name="text">
        <string>Ground Speed</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="_gps_fixtype">
       <property name="text">
        <string>Fixtype</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="_gps_lng_box"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="_gps_hAcc">
       <property name="text">
        <string>H. Accuracy</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="_mission_enable">
       <property name="text">
        <string>Mission Enabled:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="_mission_lat">
       <property name="text">
        <string>Mission Lat</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="_gps_hMSL">
       <property name="text">
        <string>hMSL</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="_gps_heading">
       <property name="text">
        <string>Heading</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="_gps_hAcc_box"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="_gps_gSpeed_box"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="_gps_heading_box"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Distance</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="_waypoint_distance_box"/>
     </item>
     <item row="11" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWebView" name="_web_view">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
     <property name="renderHints">
      <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::NonCosmeticDefaultPen|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
